### ISSUE TITLE
Videos are not playing at all

### ISSUE THREAD

**Issue by xd003** [prayag17/Blink#331]
This is my very first time using Jellyplayer. No video seem to be getting played with Jellyplayer. When i click on play button, it just keeps on loading infinitely. Morever, at this point, even the back button on top left doesn't work. i have directly exit the app altogether. The video playback is working perfectly fine on every other client - browser, jellyfin official android app/findroid & even on the jellyfin media player for desktop

 - OS: Windows 11
 - Jellyfin v10.9.1

**Comment by xd003** [comment]
> can you include the logs from the console tab and the network tab ?

Sorry but how can i access the console and network tab within JellyPlayer ? Or do you meant that inside the browser itself ? I can do that surely

**Comment by prayag17** [comment]
`Ctrl + Shift + i` opens devtools, Network tab should be present there

**Comment by xd003** [comment]
> `Ctrl + Shift + i` opens devtools, Network tab should be present there

Weirdly enough, that doesn't seem to open devtools for me when inside JellyPlayer. i guess maybe edge webview2 runtime is not present on my system ? But doen't jellyPlayer install them in case its not found. Is there a way to confirm the presence of weview runtime ?

**Comment by pypp** [comment]
> > `Ctrl + Shift + i` opens devtools, Network tab should be present there
> 
> Weirdly enough, that doesn't seem to open devtools for me when inside JellyPlayer. i guess maybe edge webview2 runtime is not present on my system ? But doen't jellyPlayer install them in case its not found. Is there a way to confirm the presence of weview runtime ?

dev tools is probably disabled in productin build.

**Comment by prayag17** [comment]
Right click should also bring up an option. Webview2 comes bundled with JellyPlayer and all windows systems have webview2 runtimes

**Comment by xd003** [comment]
> Right click should also bring up an option. Webview2 comes bundled with JellyPlayer and all windows systems have webview2 runtimes

![image](https://github.com/prayag17/JellyPlayer/assets/26503185/06b4ea7a-d9c6-45b3-8877-87d6d2d80ed5)

i get all these options on right click

**Comment by pypp** [comment]
you can also try and oepn Jellyfin server console and view the server logs. search `Jellyfin (View Console)`

**Comment by prayag17** [comment]
what version are you using @xd003?

**Comment by xd003** [comment]
> what version are you using @xd003?
JellyPlayer - v0.0.5-dev
Jellyfin - v10.9.1

**Comment by pypp** [comment]
@xd003  can you open jellyfin with logs (jellyfin (view Console)) and paste the logs you get when starting to play video from JellyPlayer ?

**Comment by xd003** [comment]
> @xd003 can you open jellyfin with logs (jellyfin (view Console)) and paste the logs you get when starting to play video from JellyPlayer ?

I am running jellyfin as a docker container, i don't see anything in jellyfin docker logs while the video in loading infinitely in JellyPlayer

**Comment by pypp** [comment]
> > @xd003 can you open jellyfin with logs (jellyfin (view Console)) and paste the logs you get when starting to play video from JellyPlayer ?
> 
> I am running jellyfin as a docker container, i don't see anything in jellyfin docker logs while the video in loading infinitely in JellyPlayer

Can you go to the jellyfin web player and go to `dashboard/logs` and then paste the logs you get.

**Comment by xd003** [comment]
> Can you go to the jellyfin web player and go to `dashboard/logs` and then paste the logs you get.
Here the the relevent lines from the log, everything above it are logs reated to the media scanning itself
```
[2024-05-18 14:45:16.426 +05:30] [INF] [151] Emby.Server.Implementations.IO.LibraryMonitor: "Movies" ("/mnt/mergerfs/Media/Movies") will be refreshed.
[2024-05-18 14:45:58.298 +05:30] [INF] [42] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "202.XXX.XX.XXX" request
[2024-05-18 14:46:51.639 +05:30] [INF] [141] Emby.Server.Implementations.IO.LibraryMonitor: "Movies" ("/mnt/mergerfs/Media/Movies") will be refreshed.
```

Here's the whole log still if you need it - https://pastebin.com/raw/Spr6h6wu

**Comment by xd003** [comment]
Still facing this issue on jellyfin 10.9.4

**Comment by MulverineX** [comment]
I'm getting this issue as well, https://github.com/prayag17/Blink/issues/335#issuecomment-2335126051

**Comment by prayag17** [comment]
hey @MulverineX, are you trying to play a series or episode?

**Comment by MulverineX** [comment]
@prayag17 an episode

**Comment by prayag17** [comment]
Yeah I fixed that a few commits back after I published v0.0.6-dev
